frontend/components/BackLink/BackLink.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { browserHistory } from "react-router";
+import Icon from "components/Icon";
+import Button from "components/buttons/Button";
+
+interface IBackLinkProps {
+  text: string;
+  path?: string;
+}
+
+const baseClass = "back-link";
+
+const BackLink = ({ text, path }: IBackLinkProps): JSX.Element => {
+  const onClick = (): void => {
+    if (path) {
+      browserHistory.push(path);
+    } else browserHistory.goBack();
+  };
+
+  return (
+    <Button className={baseClass} onClick={onClick} variant="text-icon">
```

Comment:
Just looked at this a bit more and was there a reason we decided to not use the react-router `Link` component to implement this? Have we checked that things work correctly considering we are replacing that component with the new `BackLink` component that no longer users it. I'm not sure if this will lead to issues but something to look into.

Reply:
Changing it to link, I run into that styling issue with icon being a display: flex. >.< I couldn't merge as a Link instead of a Button without the fix to .icon since it is overridden by that.